content/bibs/pubs.bib

```diff
@@ -2197,22 +2222,31 @@ @inproceedings{Vogelstein2010b_2
 
 
 
+@article{zheng2016flashmatrix,
+  title={Flashmatrix: parallel, scalable data analysis with generalized matrix operations using commodity ssds},
+  author={Zheng, Da and Mhembere, Disa and Vogelstein, Joshua T and Priebe, Carey E and Burns, R},
+  author+an={3=highlight},
+  journal={arXiv},
+  url={https://arxiv.org/abs/1604.06414},
+  volume={9},
+  pages={30},
+  year={2016},
+  keywords={tech}
+}
 
 
 
-
-
-@article {Chandrashekhar2021.01.26.428355,
-	author = {Chandrashekhar, Vikram and Tward, Daniel J and Crowley, Devin and Crow, Ailey K and Wright, Matthew A and Hsueh, Brian Y and Gore, Felicity and Machado, Timothy A and Branch, Audrey and Rosenblum, Jared S and Deisseroth, Karl and Vogelstein, Joshua T},
-	title = {CloudReg: Automatic Terabyte-Scale Cross-Modal Brain Volume Registration},
-	elocation-id = {2021.01.26.428355},
-	year = {2021},
-	doi = {10.1101/2021.01.26.428355},
-	publisher = {Cold Spring Harbor Laboratory},
-	abstract = {Quantifying terabyte-scale multi-modal human and animal imaging data requires scalable analysis tools. We developed CloudReg, an open-source, automatic, terabyte-scale, cloud-based image analysis pipeline that pre-processes and registers cross-modal volumetric datasets with artifacts via spatially-varying polynomial intensity transform. CloudReg accurately registers the following datasets to their respective atlases: in vivo human and ex vivo macaque brain magnetic resonance imaging, ex vivo mouse brain micro-computed tomography, and cleared murine brain light-sheet microscopy.Competing Interest StatementThe authors have declared no competing interest.},
-	URL = {https://www.biorxiv.org/content/early/2021/01/27/2021.01.26.428355},
-	eprint = {https://www.biorxiv.org/content/early/2021/01/27/2021.01.26.428355.full.pdf},
-	journal = {bioRxiv},
+@article{CloudReg,
+    author = {Chandrashekhar, Vikram and Tward, Daniel J and Crowley, Devin and Crow, Ailey K and Wright, Matthew A and Hsueh, Brian Y and Gore, Felicity and Machado, Timothy A and Branch, Audrey and Rosenblum, Jared S and Deisseroth, Karl and Vogelstein, Joshua T},
+    author+an = {1=trainee,12=highlight},
+    title = {CloudReg: automatic terabyte-scale cross-modal brain volume registration},
+    journal = {Nature Methods},
+    year = {2021},
+    month = {7},
+    issn = {1548-7105},
+    doi = {10.1038/s41592-021-01218-z},
+    url = {https://doi.org/10.1038/s41592-021-01218-z},
+    eprint = {https://www.nature.com/articles/s41592-021-01218-z.pdf},
     keywords = {tech}
 }
```

Comment:
`CloudReg` already exists as `peer-reviewed`, so this additional entry could be deleted?

Reply:
It seems like this tech report is peer-reviewed, so I generated two entries on purpose :)

Reply:
Would be better if keywords could be multiple items?